main/collectd/APKBUILD

```diff
@@ -1,8 +1,8 @@
 # Contributor: Łukasz Jendrysik <scadu@yandex.com>
 # Maintainer: Natanael Copa <ncopa@alpinelinux.org>
 pkgname=collectd
-pkgver=5.8.0
-pkgrel=4
+pkgver=5.8.1
+pkgrel=0
 pkgdesc="The system statistics collection daemon"
 url="http://collectd.org"
 arch="all"
```

Comment:
GPL-2.0 is not a valid SPDX identifier, aports will detect that when spdx-licenses-list